src/input.ts

```diff
@@ -52,6 +62,7 @@ export function readActionInput(): ActionInput {
   const artifactPath = core.getMultilineInput('artifact-path');
   const testCucumberPath = core.getMultilineInput('test-cucumber-path');
   const testJunitPath = core.getMultilineInput('test-junit-path');
+  const tests = YAML.parse(core.getInput('tests')) || [];
```

Comment:
> While the call of `readActionInput()` is surrounded by a `try/catch`, perhaps we should still wrap `YAML.parse` in its own `try/catch` so that the other values can still be parsed, if available

What's the motivation here? If the `tests` YAML cannot be parsed we definitely want to inform the caller that they have provided invalid input. Does it matter if we parse the other input values?

> We could also `core.warn()` in the case the input is invalid YAML.

An invalid YAML document will throw an exception when running `YAML.parse`: 

```
Uncaught YAMLParseError: Unexpected scalar at node end at line 2, column 1:

- testedItem: SAMD-45
result
^^^^^^

    at Composer.onError (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/compose/composer.js:69:34)
    at Object.resolveEnd (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/compose/resolve-end.js:31:21)
    at Object.composeDoc (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/compose/compose-doc.js:36:27)
    at Composer.next (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/compose/composer.js:149:40)
    at next (<anonymous>)
    at Composer.compose (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/compose/composer.js:131:25)
    at compose.next (<anonymous>)
    at parseDocument (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/public-api.js:45:16)
    at Object.parse (/Users/cvogt/Development/Beacon/ketryx-github-action/node_modules/yaml/dist/public-api.js:67:17) {
  code: 'UNEXPECTED_TOKEN',
  pos: [ 22, 28 ],
  linePos: [ { line: 2, col: 1 }, { line: 2, col: 7 } ]
}
```
If the YAML document has the wrong structure after parsing we'll see a typescript error.

Reply:
> What's the motivation here? If the `tests` YAML cannot be parsed we definitely want to inform the caller that they have provided invalid input. Does it matter if we parse the other input values?

That's a good point. I believe we'll still want to wrap this into a `try/catch` block and provide a meaningful error message to the user of the action, while not hiding the stack trace.

We're also in the final stretches of thinking about the newly introduced limitation of using a file only once per artifact type.